packages/js/components/src/sortable/README.md

```diff
@@ -20,8 +20,8 @@ This component accepts any valid JSX elements as children.  Adding a `key` to el
 Name | Type | Default | Description
```

Comment:
We should also update the `title` and the `usage` areas to reflect the new component name.

Reply:
Good catch!  Updated.